### ISSUE TITLE
[Eclipse+IntelliJ2017+IntelliJ2016] Docker: Can’t browse a deploment file with tomcat:8.0.20-jre8(with debugging)

### ISSUE THREAD

**Issue by bsaby** [microsoft/azure-tools-for-java#553]
Build:
artifacts/azure-toolkit-for-intellij.2016.develop.daily.11.05-23-2017.zip
artifacts/azure-toolkit-for-intellij.2017.develop.daily.11.05-23-2017.zip
artifacts/WindowsAzurePlugin4EJ.develop.daily.11.05-23-2017.zip

1. Right click project choose Publish as Docker Container...
2. Select a .WAR file to publish
![image](https://cloud.githubusercontent.com/assets/22470312/26390920/35421ade-4093-11e7-8b00-2e963b13b70e.png)
3. Predefined Docker image: tomcat:8.0.20-jre8(with debugging)
![image](https://cloud.githubusercontent.com/assets/22470312/26390927/3f83caba-4093-11e7-8ab1-a46f4e93ad0b.png)

After deployment, Click published link in Azure Activity Log dialog, the web page can be displayed successfully
![image](https://cloud.githubusercontent.com/assets/22470312/26391070/c22541a6-4093-11e7-81da-e1d65daed974.png)
![image](https://cloud.githubusercontent.com/assets/22470312/26391642/1ed6ce36-4096-11e7-9536-8c16c972d9fa.png)

Open in browse in Azure explorer,  the web page can be displayed failed.
![image](https://cloud.githubusercontent.com/assets/22470312/26391038/9ee45a74-4093-11e7-90cc-d8e9145a1b1b.png)
![image](https://cloud.githubusercontent.com/assets/22470312/26391622/195cd464-4096-11e7-9e64-178990ae1577.png)

Not regression issue.

**Comment by ZhengJun003** [comment]
According to this issue: #4353, can't reproduce this issue

**Comment by andxu** [comment]
Blocked by another issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1738797

**Comment by vscjavaci** [comment]
Close this issue since it is tracked in Azure Boards.
